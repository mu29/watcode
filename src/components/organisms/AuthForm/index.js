import React from 'react'
import styled from 'styled-components'
import { Tab, TabPage } from 'components'
import { SignInForm, SignUpForm } from 'containers'
import { palette } from 'services/style'

const StyledTab = styled(Tab)`
  margin: 1rem 0;
  border: 0.0625rem solid ${palette('gray.40')};
  background-color: ${palette('white.default')};

  @media(max-width: 768px) {
    margin: 0;
    border: none;
  }
`

const AuthForm = () => (
  <StyledTab>
    <TabPage name="로그인">
      <SignInForm />
    </TabPage>
    <TabPage name="회원가입">
      <SignUpForm />
    </TabPage>
  </StyledTab>
)

export default AuthForm
