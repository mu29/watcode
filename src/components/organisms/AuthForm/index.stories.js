import React from 'react'
import { storiesOf } from '@storybook/react'
import AuthForm from '.'

storiesOf('Organisms/AuthForm', module)
  .add('기본', () => (
    <AuthForm />
  ))
