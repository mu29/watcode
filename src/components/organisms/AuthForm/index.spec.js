import React from 'react'
import AuthForm from '.'

const wrap = (props = {}) => shallow(<AuthForm { ...props } />)

describe('<AuthForm />', () => {
  it('정상적으로 렌더링된다', () => {
    const wrapper = wrap()
    expect(wrapper).toMatchSnapshot()
  })
})
