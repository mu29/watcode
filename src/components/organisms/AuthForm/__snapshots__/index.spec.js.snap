// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AuthForm /> 정상적으로 렌더링된다 1`] = `
<StyledTab>
  <TabPage
    name="로그인"
  >
    <SignInForm />
  </TabPage>
  <TabPage
    name="회원가입"
  >
    <SignUpForm />
  </TabPage>
</StyledTab>
`;
